spec/jobs/create_derivatives_job_spec.rb

```diff
@@ -4,15 +4,13 @@
 RSpec.describe CreateDerivativesJob do
   let(:derivatives_service) { instance_double(Valkyrie::Derivatives::DerivativeService) }
   let(:file_set) { FactoryBot.create_for_repository(:file_set) }
-  let(:fixity_job) { instance_double(CheckFixityJob) }
```

Comment:
Why don't we need this anymore? And/or why did we need it initially?

Reply:
It was initially used to test that the CheckFixityJob received perform_later. This was replaced with checking for an enqueued CheckFixityJob https://github.com/pulibrary/figgy/commit/6c7b65b7f21724ec180dc07e62de9edc6d3576bd

Reply:
I see, thank you! And thanks for cleaning it up.